integration/tests/runtime-workflow-mongo.ts

```diff
@@ -42,8 +42,8 @@ beforeAll(async () => {
     graphbackApi = buildGraphbackAPI(modelText, {
       dataProviderCreator: createMongoDbProvider(db),
       plugins: [
-        new SchemaCRUDPlugin({outputPath: "./output-mongo/schema/schema.graphql"}),
-        new ClientCRUDPlugin({format: 'graphql', outputFile: './output-mongo/client/graphback.graphql'})
+        new SchemaCRUDPlugin({ outputPath: "./output-mongo/schema/schema.graphql" }),
```

Comment:
This should be gone. We do not need schema plugin

Reply:
It is needed, otherwise the integration tests fails with 
```
  ● Note 1 should be defined

    expect(received).toEqual(expected) // deep equality

    Expected: {"comments": [{"description": "Note A Comment Description", "id": "5ef5b6951f82b90668f8da94", "text": "Note A Comment"}, {"description": "Note A Comment Description", "id": "5ef5b6951f82b90668f8da95", "text": "Note A Comment 2"}], "description": "Note A Description", "id": "5ef5b6951f82b90668f8da90", "title": "Note A"}
    Received: null

      287 |   expect(response.data).toBeDefined();
      288 |   const notes = response.data.getNote;
    > 289 |   expect(notes).toEqual({
          |                 ^
      290 |     id: notesId[0],
      291 |     title: 'Note A',
      292 |     description: 'Note A Description',

      at tests/runtime-workflow-mongo.ts:289:17
      at fulfilled (../node_modules/tslib/tslib.js:112:62)

  ● Should create a new Note

    expect(received).toBeDefined()

    Received: undefined

      375 | test('Should create a new Note', async () => {
      376 |   const response = await createNote(client, { title: 'New note', description: 'New note description' });
    > 377 |   expect(response.data).toBeDefined();
          |                         ^
      378 |   expect(response.data.createNote).toEqual({ id: response.data.createNote.id, title: 'New note', description: 'New note description' });
      379 | })
      380 | 

      at tests/runtime-workflow-mongo.ts:377:25

```

Reply:
Looks like we found bug that is not related to this PR